<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alphabet</class>
 <widget class="QDialog" name="alphabet">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alphabet - Word Search Creator</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QComboBox" name="Alphabetcbox"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Letters used to fill wordsearch:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="AlphabetBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Make: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MakeUpper">
       <property name="text">
        <string>Uppercase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="MakeLower">
       <property name="text">
        <string>Lowercase</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="GetFromWeb">
       <property name="text">
        <string>Get Alphabets</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>alphabet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
